src/community-guidelines.twig

```
@@ -0,0 +1,115 @@
+						<li>Usernames that are nothing but a bunch of symbols that are impossible to pronounce aren’t very conducive of a good conversation environment. A lot of members do this to put themselves at the top of the userlist, but never talk, and that’s not really a big deal or worth the effort. Users who are talking though should have reasonable usernames. However, this does not apply to usernames in different languages and alphabets. There is a difference between having a username in Japanese and having only random symbols as your username.</li>
+					</ol>
+				</li>
+				<li>Pinging server staff, moderators, or contributors.
+					<ol type="a">
+						<li>Basically anyone with a role is a target for pinging, and the larger Paper gets, the more of an issue this becomes.</li>
+						<li>Someone pinging a particular person because they are responding to that person is not a problem and not even warranting of a warning. Someone pinging a project developer or contributor just because they want an answer in <code>#paper-help</code> is what is being addressed here.</li>
+						<li>There are some users who don’t want to be pinged. Please respect that decision.</li>
+					</ol>
+				</li>
+			</ol>
+		</p>
+	  </div>
+	  <div id="content" class="row">
+		<h3 id="zero-tolerance-items">Zero Tolerance Items</h3>
+		<p>Particularly severe issues represent a behavior and a kind of person we don’t want associated with Paper in any way. These behaviors will result in an immediate ban without warning:
+			<ul>
+				<li>Racial slurs or racist speech</li>
+				<li>Sexist slurs or sexist speech</li>
+				<li>Homophobia, Transphobia, other related hate speech</li>
+				<li>Hate speech in general</li>
```

As per DarkEyeDragon's comment in the Discord guild, this could probably be combined with the previous sentence into something like "Homophobia, transphobia, or general hate speech". That way you don't mention "hate speech" twice.

---

(This is really minor, but hear me out) I think we should keep them separate:

 * Racist slurs or racist speech
 * Sexist slurs or sexist speech
 * Homophobic or transphobic slurs or speech
 * Ableist slurs or speech
 * Any other hate speech not already listed
 
That way homophobia and transphobia aren't part of the catch-all, showing they aren't less important than the other items listed. Homophobia and transphobia work grouped together I think since they typically come one with the other, and are both LGBT issues, but grouping them with the catch-all clause might be viewed as minimizing it.

Side note, when I originally wrote this I wasn't sure how to express both "slurs or speech" without too much repetition, though I think it's good to mention both explicitly. Not sure what the best approach to that is.

Also moving the ableism bullet point above the catch-all hate speech item seems better, since doxxing and personal attacks are totally separate issues.

---

Done

---

I definitely think your point regarding not making homophobia and transphobia part of the catch-all is valid, however when reading the bullet points after one another you still have the repetitive "hate speech" in both points. In my opinion, this is a bit awkward since when reading this in my head I'm repeating this twice. I think rewording this slightly could make it flow better. For example, the first bullet point could become something along the lines of: "Homophobic, transphobic or other related slurs and messages"; or, if you're okay with combining homophobia and transphobia into one, "LGBT+ related hatred". Feel free to change this up of course, main point being is that you no longer read a duplicate "hate speech" when reading the bullet points after each other, which in my opinion improves the flow of the points.
It is also possible to reword the catch-all bullet point, however I find it difficult to come up with something. The easy route would be to move the subject around to something like "Other forms of hate speech" or "General hate speech not explicitly listed", yet I think this should be avoided. All other points list their main point as the first word in the sentence ("Racial" for racism; "Sexist" for sexism; "Homophobia, transphobia" for homophobia and transphobia) so if we move the main point (hate speech) to the back of the sentence, this would remove the continuity in the general style of the points.

Just some suggestions, for what is pretty minor anyway :)